services/core-web/src/components/Forms/AddMineWorkInformationForm.js

```diff
@@ -102,11 +83,5 @@ AddMineWorkInformationForm.defaultProps = defaultProps;
 export default compose(
   connect((state) => ({
     formValues: getFormValues(FORM.ADD_MINE_WORK_INFORMATION)(state) || {},
-  })),
-  reduxForm({
-    form: FORM.ADD_MINE_WORK_INFORMATION,
-    onSubmitSuccess: resetForm(FORM.ADD_MINE_WORK_INFORMATION),
```

Comment:
I left it in in a few places, but I suspect that this `resetForm` function is some vestigial code. I could not perceive any difference when I deleted it from a bunch of places. Consider trying to remove it from remaining areas if you're ever modifying a form!